java/ql/lib/ext/org.apache.commons.cli.model.yml

```diff
@@ -0,0 +1,6 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: sinkModel
+    data:
+      - ["org.apache.commons.cli", "DefaultParser", True, "parse", "(Options,String[])", "", "Argument[1]", "path-injection", "ai-manual"]
```

Comment:
This is a CLI options parser, I don't think the arguments are (necessarily) used to access files. At most, we could consider these local sources I guess.